README.md

```diff
@@ -1002,10 +1002,11 @@ Using git reset it is then possible to change master back to the commit it was b
 * [git-tips](https://github.com/git-tips/tips) - Small git tips
 * [git-town](https://github.com/Originate/git-town) - Generic, high-level Git workflow support! http://www.git-town.com
 
-## GUI Clients
+## GUI/TUI Clients
```

Comment:
Changes here should also be added in the Table of Contents. I'm not sure this change is necessary - just remove it?

Reply:
Indeed, I have removed this addition